projekt-03-kontakty-main/projekt-03-kontakty-main/style.css

```diff
@@ -8,3 +8,44 @@ body {
 }
 
 /* ---------------------------------------------------- */
+
+.container {
+	display: inline-flexbox;
+	flex-wrap: wrap;
+	justify-content: center;
+	align-content: center;
+	background-color: whitesmoke;
+	border-radius: 1rem;
+	align-items: stretch;
+}
+
+.item {
+	display: inline-block;
+	max-width: 22%;
+	padding: 2rem;
+	margin: 2rem;
+	background-color: white;
+	border-radius: 1rem;
+	text-align: center;
+}
+
+.foto {
+	max-width: 150px;
+	height: auto;
+	border-radius: 5rem;
+}
+
+.zapati {
+	display: inline-flexbox;
+	align-content: center;
+	word-spacing: 1rem;

```

Comment:
Tady bych nenastavovala odsazení pomocí `word-spacing`, i když je to dobrý hack a funguje ti to. Ale úplně to neodpovídá významu, kterému má tahle vlastnost sloužit - mezery mezi slovy. Použila bych klasicky `margin`